src/libraries/System.Net.Http/src/System.Net.Http.csproj

```diff
@@ -31,21 +31,29 @@
   <PropertyGroup>
     <ILLinkDirectory>$(MSBuildThisFileDirectory)ILLink\</ILLinkDirectory>
   </PropertyGroup>
+  <!-- DesignTimeBuild requires all the TargetFramework Derived Properties to not be present in the first property group. -->
+  <PropertyGroup>
+    <TargetPlatformIdentifier>$([MSBuild]::GetTargetPlatformIdentifier('$(TargetFramework)'))</TargetPlatformIdentifier>
```

Comment:
Can we or is it worth to this globally for all projects that multitarget or is that an expensive call?

Reply:
The SDK already does this but in the TargetFrameworkInference.targets file which means the property won't be for properties inside the project file. Because of that, we are setting it ourselves in exactly the same way the SDK sets it and the result will be cached and the duplicate is avoided.

We can't set this globally as it relies on the TargetFramework prop which isn't available at that point.